sklearn/feature_selection/from_model.py

```diff
@@ -111,3 +141,118 @@ def transform(self, X, threshold=None):
             return X[:, mask]
         else:
             raise ValueError("Invalid threshold: all features are discarded.")
+
+
+class SelectFromModel(BaseEstimator, SelectorMixin):
+    """Meta-transformer for selecting features based on importance weights.
+
+    Parameters
+    ----------
+    estimator : object
+        The base estimator from which the transformer is built.
+        This can be both a fitted (if ``prefit`` is set to True)
+        or a non-fitted estimator.
+
+    threshold : string, float, optional default None
```

Comment:
I think this is one of those cases where `default None` is not very helpful, but the description should say "By default, [this is its behaviour]..."